inst/tests/tests.Rraw

```diff
@@ -12712,9 +12712,9 @@ test(1899.18, as.matrix(DT, rownames=TRUE, rownames.value=1:nrow(DT)), error="ro
 
 # index argument for fread, #2633
 DT_str = c('a,b\n3,1\n2,2\n1,1\n2,1\n3,2')
-test(1900.1, attributes(attr(fread(DT_str, index = 'a'), 'index')),
+test(1900.1, lapply(attributes(attr(fread(DT_str, index = 'a'), 'index')), c),
```

Comment:
don't expect reader to poke around for other tests doing this, easy to miss the same being done (currently) 40 lines earlier

```suggestion
test(1900.1, lapply(attributes(attr(fread(DT_str, index = 'a'), 'index')), c), ## lapply(, c) to ensure no starts, maxgrpn attributes
```